### PR TITLE
[PW 2234]: Configuration panel header

### PR DESCRIPTION
**Pull request by acampos1916**
Here we're adding a header with useful links to the module's configuration panel

### CODE REVIEW

File: views/templates/front/get-started.tpl [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ *}
+
+<div style="display: flex;">
+    <div style="float:left;padding-right:10px;">
+        <img width="120px" src="{$logo}">
+    </div>
+    <div>
+        <p>Adyen all-in-one payments platform.</p>
+        <p>Offer key payment methods anywhere in the world at the flick of a switch.</p>
+        <p>Easy integration with our in-house built Prestashop Plugin, no set-up fee.</p>
```
Comment (Aleffio) [inline views/templates/front/get-started.tpl:31]:
```suggestion
        <p>Easy integration with our in-house built PrestaShop Plugin, no set-up fee.</p>
```

File: adyenofficial.php [hunk 2]
```diff
@@ -92,6 +92,30 @@ class AdyenOfficial extends PaymentModule
      */
     private $adyenPaymentResponseModel;
 
+    /**
+     * Links to show on the Adyen payment method header
+     *
+     * @var array
+     */
+    private $links = array(
```
Comment (cyattilakiss) [inline adyenofficial.php:100]: Is it better maybe if static?

File: adyenofficial.php [hunk 3]
```diff
@@ -92,6 +92,30 @@ class AdyenOfficial extends PaymentModule
...
+            "label" => "Support",
+            "url" => "https://support.adyen.com/hc/en-us/requests/new?ticket_form_id=78764"
+        ),
+        array(
+            "label" => "GitHub",
+            "url" => "https://github.com/Adyen/adyen-prestashop/releases"
+        ),
+        array(
+            "label" => "PrestaShop Marketplace",
+            "url" => "https://addons.prestashop.com"
```
Comment (rikterbeek) [inline adyenofficial.php:115]: shall we redirect to our marketplace page ?

File: views/templates/front/get-started.tpl [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+ *}
+
+<div style="display: flex;">
+    <div style="float:left;padding-right:10px;">
+        <img width="120px" src="{$logo}">
+    </div>
+    <div>
+        <p>Adyen all-in-one payments platform.</p>
+        <p>Offer key payment methods anywhere in the world at the flick of a switch.</p>
+        <p>Easy integration with our in-house built PrestaShop Plugin, no set-up fee.</p>
```
Comment (rikterbeek) [inline views/templates/front/get-started.tpl:31]: Let's explain they need an account on Adyen. They can create this account on  https://www.adyen.com/signup

Comment (acampos1916) [reply views/templates/front/get-started.tpl:31]: Added the signup link with the wording used in most documentation (https://docs.adyen.com/classic-integration/3d-secure/3d-secure-1/3d-secure-1-authentication-only)

File: adyenofficial.php (hunk 3 above)
Comment (acampos1916) [reply adyenofficial.php:115]: At the moment we don't have a marketplace page, will set a reminder to update this when we do

**Pull request merged**